hokusai/lib/common.py

```diff
@@ -380,3 +380,28 @@ def write_temp_file(data, dir):
 def yaml_content_with_header(content_string):
   ''' return content with yaml header prepended '''
   return YAML_HEADER + content_string
+
+def confirm_delete_resources(project_name, environment, yaml_file):
+  ''' Display confirmation prompt for destructive delete operations '''
+  import click
```

Comment:
is there a reason to inline the import here rather than put at the top of the file?

Reply:
Ahh, good catch – I've gone ahead and moved it to the top of the file.